collect.py

```diff
@@ -7,6 +7,18 @@
 from selenium import webdriver
 from selenium.common.exceptions import NoAlertPresentException, NoSuchWindowException, TimeoutException
 
+from sys import platform as _platform
+if _platform == "linux" or _platform == "linux2":  # linux
+    chromebin = "tools/chrome-linux/chrome"
+    nightybin = 'tools/nightly/firefox-bin'
+
+
+elif _platform == "darwin":    # MAC OS X
+    chromebin = "tools/chrome.app/Contents/MacOS/chrome"
+    nightybin = 'tools/Nightly.app/Contents/MacOS/firefox'
+
+if not os.path.exists('data'):
```

Comment:
Use this snippet instead, which avoids the existance check:
```
    try:
        os.mkdir('data')
    except OSError:
        pass
```

It would be great if you could add a `mkdir` function to utils.py that does this and use it both here and in `utils.prepare_images`.